lib/assets/javascripts/utils/paginable.js

```diff
@@ -7,7 +7,13 @@ $(() => {
     // Note we update ONLY .paginable-results container although the AJAX response
     // returns the entire paginable layout html. That permits, keeping any value entered
     // in the search box (if any)
-    $(e.target).closest(paginableSelector).find('.paginable-results').html($(data).find('.paginable-results').children());
+    const paginable = $(e.target).closest(paginableSelector);
+    paginable.find('.paginable-results').html($(data).find('.paginable-results').children());
+    // Update the serahc box's action so that it retains sort settings
+    const searchPath = paginable.find('.paginable-results #search-path');
```

Comment:
html id are unique across an entire document, therefore .paginable-results can be omitted. Anyway, I don't seem to see #search-path id associated to any html attribute. Do am I missing anything here?

Reply:
its a new hidden input field that is being used to retain the sort rules after the paginable-results is refreshed. Currently, the sort field/direction are lost anytime the user searches.